### DISCUSSION TITLE
Support publishing actions from monorepos

### DISCUSSION THREAD

**Discussion by misterquestions** [community/community#42674] [Actions]
### Select Topic Area

General

### Body

Hey there!

Recently I've been working in a monorepo with actions, all seems fine since you can use them as long as they have a release and its a public repository.

However, I'd like to make them available to the marketplace so anyone who wants to use them can search for them easily.

The issue here is that GitHub only supports publishing actions to the marketplace when they exist in a repo that only contains that single action files.

Is there any planned support for this?

**Comment by xhyrom** [comment]
that would be good
